hi-ml/testhiml/testhiml/test_run_ml.py

```diff
@@ -66,12 +71,15 @@ def ml_runner_with_container() -> Generator:
 
 
 @pytest.fixture()
-def ml_runner_with_run_id() -> Generator:
+def ml_runner_with_src_ckpt() -> Generator:
+    """
+    A fixture with a training runner for the HelloWorld model that has a src_checkpoint set.
```

Comment:
can we give this fixture a more specific name, src_ckpt can be one of (run_id, local path, url) hence the previous name